exchanges/btse/btse_wrapper.go

```diff
@@ -320,23 +351,22 @@ func (b *BTSE) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([]e
 			Status:    order.Status,
 		}
 
-		fills, err := b.GetFills(order.ID, "", "", "", "")
+		fills, err := b.GetFills(order.ID, "", "", "", "", "")
 		if err != nil {
 			log.Errorf("unable to get order fills for orderID %s", order.ID)
 			continue
 		}
 
-		for i := range *fills {
-			f := (*fills)[i]
-			createdAt, _ := time.Parse(time.RFC3339, f.CreatedAt)
+		for i := range fills {
+			createdAt, _ := time.Parse(time.RFC3339, fills[i].CreatedAt)
 			openOrder.Trades = append(openOrder.Trades, exchange.TradeHistory{
 				Timestamp: createdAt,
-				TID:       f.ID,
-				Price:     f.Price,
-				Amount:    f.Amount,
+				TID:       fills[i].ID,
+				Price:     fills[i].Price,
+				Amount:    fills[i].Amount,
 				Exchange:  b.Name,
-				Type:      exchange.OrderSide(f.Side).ToString(),
-				Fee:       f.Fee,
+				Type:      exchange.OrderSide(fills[i].Side).ToString(),
```

Comment:
You are casting a string to an `exchange.Orderside` which is a custom type of string, and then casting it as a string. You can just keep it as `fills[i].Side,` and save on casting.